orbit/pkg/setup_experience/setup_experience.go

```diff
@@ -392,3 +400,290 @@ func WriteSetupExperienceStatusFile(rootDir string, exp *SetupExperienceInfo) er
 	}
 	return nil
 }
+
+func (s *SetupExperiencer) RunLegacy(oc *fleet.OrbitConfig) error {
```

Comment:
This is just the currently-published `Run` method: https://github.com/fleetdm/fleet/blob/rc-patch-fleet-v4.75.1/orbit/pkg/setup_experience/setup_experience.go#L54